exonum/src/node/mod.rs

```diff
@@ -175,6 +177,7 @@ impl Default for EventsPoolCapacity {
             network_events_capacity: 512,
             timeout_requests_capacity: 128,
             api_requests_capacity: 1024,
+            internal_events_capacity: 128,
```

Comment:
Please add `BREAKING CHANGE` badge. And add comment about these parameter to changelog.